### PR TITLE
Messaging transport version

### PR DESCRIPTION
**Pull request by khdevel**
[messaging_transport_version]
    - new parameter 'messaging_transport_version'
    - it is related to the issue that when $version is set to e.g. 2.12.0-1.el7 it is also assigned
    to packages like 'python-gofer-qpid', 'python-gofer-amqp' or 'gofer' which do not have such number
    
     Changes to be committed:
    - modified:   manifests/init.pp
    - modified:   manifests/params.pp
    - modified:   manifests/install.pp
    - modified:   manifests/consumer.pp
    - modified:   manifests/consumer/install.pp
    - modified:   manifests/consumer/params.pp

### CODE REVIEW

File: manifests/consumer.pp [hunk 1]
```diff
@@ -88,6 +88,8 @@
 #
 # $messaging_transport::           The AMQP transport name. Valid options are 'qpid' or 'rabbitmq'. The default is 'qpid'.
 #
+# $messaging_version::             Determines the version of packages related to the 'messaging transport protocol'.
```
Comment (ekohl) [inline manifests/consumer.pp]: You're documenting a parameter that doesn't exist in this class.

Comment (khdevel) [inline manifests/consumer.pp]: I have updated the code...

**Comment by ehelms** [comment]
@ekohl They come from the gofer package so they should be in sync -- here is an example http://koji.katello.org/koji/buildinfo?buildID=30583

**Comment by ekohl** [comment]
@ehelms do you then agree this can be merged?

**Comment by ekohl** [comment]
Thanks!

**Comment by khdevel** [comment]
Thank you very much!

**Pull request merged**